### PR TITLE
Improve unit test cases coverage

### PR DESCRIPTION
**Pull request by EricBlack**
- [x] Improve contract test cases.
- [x] Fix one issue about update calculate coefficient.

### CODE REVIEW

**Comment by loning** [comment]
do not make a big PR, one work one PR

**Comment by EricBlack** [comment]
> do not use any editor formatter
> only keep your chagnes

ok, got it.

**Comment by EricBlack** [comment]
> do not make a big PR, one work one PR

ok, this pr is try to improve contract test cases, will create new PR for other test cases.

File: test/AElf.Contracts.Parliament.Tests/ParliamentContractTest.cs [hunk 1]
```diff
@@ -753,7 +781,7 @@ private async Task<Hash> CreateProposalAsync(ECKeyPair proposalKeyPair, Address
                 Symbol = "ELF",
                 Amount = 100,
                 To = Tester,
-                Memo = "Transfer"
+                Memo = Guid.NewGuid().ToString()
```
Comment (rosona) [inline test/AElf.Contracts.Parliament.Tests/ParliamentContractTest.cs]: It's better not use the random string.

Comment (EricBlack) [reply test/AElf.Contracts.Parliament.Tests/ParliamentContractTest.cs]: > It's better not use the random string.

Add random string just for different proposal tx, have been add comments for it.

**Pull request merged**